ETC.tsv

```diff
@@ -3811,7 +3811,7 @@ ETC_20150317_003810	Quest Starting Map:
 ETC_20150317_003811	Start NPC:
 ETC_20150317_003812	$ activated {nl}
 ETC_20150317_003813	 Use this to warp anywhere!
-ETC_20150317_003814	This camp does not have an activated Goddess Statue.{nl} Therefore you cannot move.
+ETC_20150317_003814	{memo Check if this appears in any other scenario. Don't put a $ here, changes linebreak.}You have not activated any other Goddess Statue, so you cant teleport yet.
```

Comment:
@imcgames: why is the message saying that in the first place? The translation is correct, but the message makse no sense whatsoever. The player walks up to the statue, it lights up, they speak to it and the message tells them the statue is not activated? Why? It just lit up. 

My version assumes that the player has activated the statue but since there is no other one you can port to, you are unable to port. Is that the intended meaning?